src/components/calcite-chip/calcite-chip.scss

```diff
@@ -1,3 +1,15 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-chip-spacing-unit-l: adds horizontal padding before chip text
+ * @prop --calcite-chip-spacing-unit-s: adds horizontal padding after chip text
+ * @prop --calcite-chip-button-border-radius: sets border-radius on close button pseudo-class colors (:hover, :active, :focus)
```

Comment:
These should all be internal, unsure about the hover / press ones, maybe @bstifle knows if those should be overrideable.

Reply:
The `-hover`/`-press` CSS vars come from the global theme, so I'l leave those as-is here (ie., they're override-able).

Reply:
Yeah, I guess I just meant, should those just be overridable with the global theme css vars, not specific tree ones? Maybe more of a css refactor within Tree to use those.

Reply:
gotcha, @macandcheese I'll remove these

Reply:
done!